test/live_view_todo_web/live/page_live_test.exs

```diff
@@ -17,20 +17,17 @@ defmodule LiveViewTodoWeb.PageLiveTest do
   test "toggle an item", %{conn: conn} do
     {:ok, item} = Item.create_item(%{"text" => "Learn Elixir"})
     assert item.status == 0
-
     {:ok, view, _html} = live(conn, "/")
-    assert render_click(view, :toggle, %{"id" => item.id, "value" => 1}) =~ "completed"
 
-    updated_item = Item.get_item!(item.id)
-    assert updated_item.status == 1
+    assert view |> element("#item-#{item.id}") |> render_click()
   end
 
   test "delete an item", %{conn: conn} do
     {:ok, item} = Item.create_item(%{"text" => "Learn Elixir"})
     assert item.status == 0
 
     {:ok, view, _html} = live(conn, "/")
-    assert render_click(view, :delete, %{"id" => item.id}) =~ "Todo"
+    view |> element("#delete-item-#{item.id}") |> render_click()
```

Comment:
did you mean to remove the `assert` here?

Reply:
I can add back the assert :+1:

Reply:
If it's not needed, fair enough. but if we aren't _assigning_ the result of this sequence to anything, 
then it's _unclear_ why it's there. 🤷‍♂️ 
So it _either_ needs a comment to the effect of:
```elixir
# Render the initial view then click on the item by id to delete it:
```
***or*** leave the `assert`. 😉